nextcord/ext/help_commands/embedded.py

```diff
@@ -0,0 +1,138 @@
+from typing import Mapping, Union
+
+from nextcord.ext.commands import Cog, Command, Group, HelpCommand
+
+from nextcord import Embed, Message
+
+
+class EmbeddedHelpCommand(HelpCommand):
+    """Custom help command override using embeds"""
+
+    def __init__(self, **options):
```

Comment:
Can these be explicit rather then **kwargs as we are trying to move away from hiding that info + it provides a smoother experience for our end users.

Note, this also means that params could be typehinted for further clarity